Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/instructionsearch/ui/SearchDirectionWidget.java

```diff
@@ -41,7 +41,6 @@
 
 	/**
 	 * 
-	 * @param plugin
 	 * @param title
 	 * @param dialog
 	 */
```

Comment:
May you provide the documentation here

Reply:
I have no clue what should be here, I just saw that what currently is there doesn't make sense and corrected it to a slightly better version which at least doesn't create wrong links